packages/amplication-data-service-generator/src/server/create-dotenv.template.env

```
@@ -4,3 +4,5 @@ POSTGRESQL_PORT=${dbPort}
 POSTGRESQL_URL=postgres://${dbUser}:${dbPassword}@${dbHost}:${dbPort}${dbName}
 BCRYPT_SALT=10
 COMPOSE_PROJECT_NAME=amp_${appId}
+JWT_SECRET_KEY="Change_ME!!!"
+JWT_EXPIRATION="2d" # more options in https://github.com/nestjs/jwt#jwtservicedecodetoken-string-options-decodeoptions-object--string
```

keep the current standard and remove the quotes.
```suggestion
JWT_SECRET_KEY=Change_ME!!!
JWT_EXPIRATION=2d 
```